Testnet/KeyValue/KeyValue/KeyValue.cs

```diff
@@ -0,0 +1,36 @@
+using System;
+using Stratis.SmartContracts;
+
+public class KeyValueContract : SmartContract
+{
+    public string Get(Address address, string key)
+    {
+        return State.GetString($"{address}:{key}");
+    }
+
+    public void Set(string key, string value)
+    {
+        Assert(key != null, "Key cannot be null");
+
+        Assert(value != null, "Value cannot be null");
+
+        Assert(key.Length <= this.MaxLength, "Key length cannot exceed maximum");
+
+        Assert(value.Length <= this.MaxLength, "Value length cannot exceed maximum");
+
+        State.SetString($"{Message.Sender}:{key}", value);
```

Comment:
This is a nice to have contract, could be used for a lot of things.

Only suggestion is to add a log here to track historical changes through receipts.